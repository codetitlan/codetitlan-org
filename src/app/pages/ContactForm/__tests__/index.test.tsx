import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { ContactForm } from '../index';

const renderer = createRenderer();

describe('<ContactForm />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<ContactForm />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
