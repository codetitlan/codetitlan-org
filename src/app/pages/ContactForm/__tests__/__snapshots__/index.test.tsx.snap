// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactForm /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      Codetitlan 
    </title>
    <meta
      content="A Boilerplate application homepage"
      name="description"
    />
  </Helmet>
  <form
    onSubmit={[Function]}
  >
    <input
      defaultValue="test"
      name="example"
      onBlur={[Function]}
      onChange={[Function]}
    />
    <input
      name="exampleRequired"
      onBlur={[Function]}
      onChange={[Function]}
    />
    <ForwardRef
      type="submit"
    >
      Submit
    </ForwardRef>
  </form>
</React.Fragment>
`;
